src/functionalTest/resources/scenarios/RIA-2374-update-hearing-adjustments-validation.json

```diff
@@ -1,5 +1,5 @@
 {
-  "description": "RIA-2374 update hearing adjustments errors when 'updatedHearingRequirements' is not available",
+  "description": "RIA-2374 update hearing adjustments errors when updated Hearing Requirements are not available",
```

Comment:
Possibly worth including your ticket no RIA-2398 in the description (same format as the other tests).